information_for_authors.md

```diff
@@ -0,0 +1,11 @@
+# Using LiveCoMS templates for your own living document
+
+## Getting started
+
+To get started with our template, clone [this repository](https://github.com/livecomsjournal/article_templates) by clicking the "clone or download button" and download a `.zip` file. 
+Then navigate to the `templates` directory, copy the files to where you will be working, and get started!
```

Comment:
These instructions seem unclear -- we have an "after forking" section but we don't tell them to fork. And are we proposing two models here? (Getting started without forking, or getting started via forking?)

It seems like we want to just write step-by-step instructions. I think these should also include (a) first forking, then (b) retrieving exactly the files they want (e.g. the specific template for best practices documents or some such), (c) removing unnecessary files, (d) moving the `your.README.md` as you describe, then (e) editing it.

Probably they should also get a default license when they fork; we should make sure it is the one we want as default (CC-BY), but add a note pointing out they can update if they want...